rigex-front-end/src/pages/Rig/Rig.js

```diff
@@ -29,6 +29,16 @@ const Rig = ({
     return null;
   });
 
+  const checkForDrilling = (status) => {
+    if (status === 'drilling') {
+      const drillingWell = listOfWells.find((well) => well.status === status);
```

Comment:
you could just use `return !listOfWells.some((well) => well.status === status)`

Reply:
This will return false if the `.some` returns true, which I believe is the logic you are working with below

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some